module-2/backend/week_one.md

```diff
@@ -1,32 +1,85 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET, to retrieve information
+POST, to send or upload information
+PUT, to write over existing data
+PATCH, to edit existing data
+DELETE, to delete data
+
 2. What is Sinatra?
+It's a framework for web applications. This means that it's a software written in Ruby that responds to the Ruby language. The software has its own code and methods that describe and create ways to structuring code, files, and folders so that, if the structure is followed, the methods will perform tasks that would otherwise take TON of ruby code to reinvent.
+
 4. What is MVC?
+Model, View, Controller. This is a modern structure that is followed by most web platforms where you separate functionality into like files and folders. The controller(s) holds routes for the pages a user visits. The model(s) hold any necessary code or logic that is needed for a class and its instances, and the view(s) and are the pages that a user is presented with. The views are populated with logic as defined in the models and/or the view itself (using erb files instead of plain html files in our case).
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+There are methods that rely on certain parts of our file structure and names in order to work correctly. If Sinatra is running a method that looks for a file with a certain name or location but can't find it, the expected outcome may not be what you anticipated, or you might get an error.

```

Comment:
Good point! Also helps people unfamiliar with your project to be able to jump in and contribute quickly.